app/javascript/components/BubbleChart.js

```diff
@@ -1,13 +1,53 @@
 import React from "react"
 import PropTypes from "prop-types"
-class BubbleChart extends React.Component {
-  render () {
-    return (
-      <React.Fragment>
-        Bubble Chart to come
-      </React.Fragment>
-    );
-  }
+import { VictoryScatter, VictoryTooltip, VictoryGroup } from "victory";
+
+const createRow = (dataRow, bubbleColor) => {
+  return (
+    <>
+      <div>
+        title
+      </div>
+      {dataRow.map(el => ( // = dataRow
+        <div className="bubble-chart__cell" key={el}>
+          <VictoryGroup style={{ height: 'auto' }}>
+            <VictoryScatter
+              style={{
+                data: { fill: bubbleColor }, labels: { fill: bubbleColor }
+              }}
+              bubbleProperty="amount"
+              maxBubbleSize={50}
+              minBubbleSize={15}
+              labels={({ datum }) => `Y: ${datum.y}`}
+              labelComponent={<VictoryTooltip flyoutWidth={200} flyoutHeight={100} style={{ fontSize: '52px' }} />}
+              data={[
+                {x: 1, y: -4, amount: 30 },
+                {x: 2, y: 4, amount: 40},
+                {x: 3, y: 2,  amount: 25},
+                {x: 4, y: 1, amount: 45}
+              ]}
+            />
+          </VictoryGroup>
+        </div>
+      ))}
+    </>
+  )
+}
+
+const BubbleChart = () => {
+  return (
+    <React.Fragment>
```

Comment:
this react fragment is not needed here, right?

Reply:
true - removed it